tests/unit/data-access/FetchParseValueRepository.spec.ts

```diff
@@ -0,0 +1,119 @@
+import FetchParseValueRepository from '@/data-access/FetchParseValueRepository';
+import TechnicalProblem from '@/data-access/errors/TechnicalProblem';
+import PrecisionError from '@/data-access/errors/PrecisionError';
+
+describe( 'FetchParseValueRepository', () => {
+
+	it( 'parses values with time datatype', async () => {
+		const testEndpoint = 'https://example.com/w/api.php';
+		const repo = new FetchParseValueRepository(
+			testEndpoint,
+		);
+		const value = '1994-02-08';
+		const datatype = 'time';
+		const expectedResult = {
```

Comment:
That looks like the thing that the Repository is supposed to return, but it seems to also be used in line 28 as the mocked API response?

Reply:
fixed